### PR TITLE
Hide survey fields on EventCreate form and EventDetail view

### LINKED ISSUE #944: Hide section for survey results
We've used a couple of different survey tools - this was requested in an earlier iteration of survey tools.  We're not recording these here now so I'd like to hide this from the view until we decide what to do.

### PR DESCRIPTION
**Pull request by narayanaditya95**
Fixes #944

### CODE REVIEW

**Comment by pbanaszkiewicz** [comment]
Should we perhaps comment out instead of deleting?

**Comment by narayanaditya95** [comment]
I am in favour of re-implementing it as accordions so that they can be revealed on demand.
Crispy form has in-built support for accordions.[1]

[1]: http://django-crispy-forms.readthedocs.io/en/latest/layouts.html

**Comment by pbanaszkiewicz** [comment]
Alright, let's go this way. Please leave a link to accordions somewhere in the code (perhaps in Event model near the field declarations?).

**Comment by narayanaditya95** [comment]
@pbanaszkiewicz 
I rewrote the crispy implementation. It does not make any cosmetic change to the form rendering apart from adding the ability to hide the `Location details` accordian.

File: workshops/forms.py [hunk 1]
```diff
@@ -393,17 +394,7 @@ def clean_end(self):
...
-                  'assigned_to', 'tags', 'url', 'language', 'reg_key',
-                  'admin_fee', 'invoice_status', 'attendance', 'contact',
-                  'notes', 'country', 'venue', 'address', 'latitude',
-                  'longitude', 'learners_pre', 'learners_post',
-                  'instructors_pre', 'instructors_post', 'learners_longterm')
-
-        # WARNING: don't change put any fields between 'country' and
-        #          'longitude' that don't relate to the venue of the event
-
+        fields = '__all__'
```
Comment (narayanaditya95) [inline workshops/forms.py]: The field order is now determined by `self.helper.layout`.

File: workshops/forms.py [hunk 2]
```diff
@@ -393,17 +391,10 @@ def clean_end(self):
...
+                  'assigned_to', 'tags', 'url', 'language', 'reg_key', 'venue',
                   'admin_fee', 'invoice_status', 'attendance', 'contact',
-                  'notes', 'country', 'venue', 'address', 'latitude',
-                  'longitude', 'learners_pre', 'learners_post',
-                  'instructors_pre', 'instructors_post', 'learners_longterm')
-
-        # WARNING: don't change put any fields between 'country' and
-        #          'longitude' that don't relate to the venue of the event
-
+                  'notes', 'country', 'address', 'latitude', 'longitude', )
```
Comment (narayanaditya95) [inline workshops/forms.py]: This seems like a violation of DRY. I am repeating the fields I would to display in both `helper.layout` and `Meta.fields`.

File: workshops/forms.py [hunk 3]
```diff
@@ -340,37 +341,34 @@ class EventForm(forms.ModelForm):
...
+        'url',
+        'language',
+        'reg_key',
+        'admin_fee',
+        'invoice_status',
+        'attendance',
+        'contact',
+        'notes',
+        Accordion(
+            AccordionGroup('Location details',
```
Comment (pbanaszkiewicz) [inline workshops/forms.py]: So previously we had some implementation that's now in the `Accordion`? The output look is the same -- which means less code maintenance for us (big win)!

**Pull request merged**